File: backend-app/src/main/java/tom/projects/monkeylog/dto/workout/WorkoutResponse.java
package tom.projects.monkeylog.dto.workout;

import lombok.Data;
import tom.projects.monkeylog.model.workout.Type;

import java.time.LocalDateTime;

@Data
public class WorkoutResponse {
    private Long id;
    private String name;
    private String note;
    private Type type;
    private LocalDateTime startDate;
    private LocalDateTime endDate;
}

File: backend/src/main/kotlin/tom/com/workout/logbook/dto/workout/WorkoutResponse.kt
package tom.com.workout.logbook.dto.workout

import tom.com.workout.logbook.model.workout.WorkoutType
import java.time.Instant
import java.util.*

data class WorkoutResponse(
    val id: UUID,
    val name: String,
    val note: String? = null,
    val workoutType: WorkoutType,
    val startDate: Instant? = null,
    val endDate: Instant? = null,
)
